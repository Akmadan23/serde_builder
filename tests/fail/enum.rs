use serde_builder::SerdeBuilder;

#[derive(SerdeBuilder)]
enum MyEnum {
    A,
    B,
    C
}

fn main() {}
