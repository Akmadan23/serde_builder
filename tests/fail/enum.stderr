error: proc-macro derive panicked
 --> tests/fail/enum.rs:3:10
  |
3 | #[derive(SerdeBuilder)]
  |          ^^^^^^^^^^^^
  |
  = help: message: SerdeBuilder only works for structs.
